### ISSUE TITLE
BSNet implementation

### ISSUE THREAD

**Issue by smilecode-github** [voldemortX/pytorch-auto-drive#142]
Hi.

I am interested in lane detection based on parameters such as BézierLaneNet or PolyLaneNet.

Recently, I found a new paper on parameter-based lane detection called BSNet.

The dissertation path is as follows.
BSNet: Lane Detection via Draw B-spline Curves Nearby: [https://arxiv.org/pdf/2301.06910.pdf](url)

Can you review an implementation for BSNet similar to BézierLaneNet?

**Comment by voldemortX** [comment]
@smilecode-github If you mean code review, then I would happily review any community contributions, provided with validation set tuning and avg/best numbers reported on at least one dataset.

**Comment by smilecode-github** [comment]
> @smilecode-github If you mean code review, then I would happily review any community contributions, provided with validation set tuning and avg/best numbers reported on at least one dataset.

Thank you for your answer!
The meaning of code review is a little ambiguous, so I ask for confirmation.
Do you mean reviewing the code implementation of the BSNet paper?
Thank you.

**Comment by voldemortX** [comment]
@smilecode-github FYI, my meaning is to review a PR in this repo.

**Comment by smilecode-github** [comment]
> @smilecode-github FYI, my meaning is to review a PR in this repo.

I understand what you mean.
Thank you!
